RecoVertex/BeamSpotProducer/python/BeamSpot_cff.py

```diff
@@ -1,4 +1,11 @@
 import FWCore.ParameterSet.Config as cms
 
 from RecoVertex.BeamSpotProducer.BeamSpot_cfi import *
+from RecoVertex.BeamSpotProducer.offlineBeamSpotCUDA_cfi import offlineBeamSpotCUDA
 
+offlineBeamSpotTask = cms.Task(offlineBeamSpot)
+
+from Configuration.ProcessModifiers.gpu_cff import gpu
+_offlineBeamSpotTask_gpu = offlineBeamSpotTask.copy()
+_offlineBeamSpotTask_gpu.add(offlineBeamSpotCUDA)
+gpu.toReplaceWith(offlineBeamSpotTask, _offlineBeamSpotTask_gpu)
```

Comment:
Should the `offlineBeamSpotTask` be added to `globalreco_trackingTask` already in this PR?
https://github.com/cms-sw/cmssw/blob/a743515e8f09f6f7ca669e15ed5cad9f44992ae4/Configuration/StandardSequences/python/Reconstruction_cff.py#L108

Reply:
Just to note, if it is added in this PR, then also the loading of `CUDAService` needs to be added
https://github.com/cms-patatrack/cmssw/blob/545096cbc0117c7426eaa9da81addc20fbb0c9e2/Configuration/StandardSequences/python/Services_cff.py#L15-L18